pkg/console/subresource/configmap/configmap_test.go

```diff
@@ -850,6 +855,12 @@ func testPlugins(pluginName, serviceName, serviceNamespace string) *v1alpha1.Con
 	}
 }
 
+func testPluginsI18nAnnotation(pluginName, serviceName, serviceNamespace string) *v1alpha1.ConsolePlugin {
+	plugin := testPlugins(pluginName, serviceName, serviceNamespace)
+	plugin.SetAnnotations(map[string]string{api.PluginI18nAnnotation: "true"})
```

Comment:
I would add a test for `"false"` and other values to make sure they're not added as namespaces.